samplers/samplers.go

```diff
@@ -72,6 +73,131 @@ type JSONMetric struct {
 	Value []byte `json:"value"`
 }
 
+// CardinalityCountName is the name of the metric that we eventually pass on
+// to the sinks.
+const CardinalityCountName = "cardinality.count"
+
+// CardinalityCountType is the string that denotes the type of the metric. We
+// use this when we serialize to a JSON metric.
+const CardinalityCountType = "cardinality_count"
+
+// cardinalityPrecision is the precision of the HLL that stores cardinality for
+// each metric name.
+const cardinalityPrecision = 18
```

Comment:
Should this be configurable? #189 has pointed out that we seem to use an overly "big" HLL @ precision of 18.